nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractDatabaseFetchProcessor.java

```diff
@@ -446,7 +446,7 @@ protected static String getLiteralByType(int type, String value, String database
             case TIME:
                 return "'" + value + "'";
             case TIMESTAMP:
-                if ("Oracle".equals(databaseType)) {
+                if (!StringUtils.isEmpty(databaseType) && databaseType.contains("Oracle")) {
```

Comment:
How about using `equalsIgnoreCase` in checking if databaseType contains oracle? It's trivial but a safety check, IMHO, in case some one changes the `getName` in `DatabaseAdapter` implementations.